public/data/Sejal-16.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Sejal Zambare",
+    "bio": " Hi there 😃,I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities.Feel free to connect with me.",
```

Comment:
```suggestion
    "bio": " Hi there 😃,I am in my pre-final year of computer science engineering. I am passionate about web development and competitive coding. I am now looking for open source opportunities. Feel free to connect with me.",
```

Reply:
Thanks for the suggestion :) Appreciate for reviewing.